staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -439,6 +439,13 @@ const (
 	//
 	// The annotation is added to a "Bookmark" event.
 	InitialEventsAnnotationKey = "k8s.io/initial-events-end"
+
+	// InitialEventsListGVKAnnotationKey the name of the key
+	// under which an annotation holding serialized GVK
+	// of the list is stored.
+	//
+	// The annotation is added to a "Bookmark" event.
+	InitialEventsListGVKAnnotationKey = "kubernetes.io/initial-events-list-gvk"
```

Comment:
i haven't found a helper for endoing/decoding a GVK, should we split into 2 ?

Reply:
I think that setting `kind` would be easier and this is what is missing.

Reply:
three fields, group, version, kind.